corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -373,15 +370,15 @@ hqDefine("cloudcare/js/form_entry/form_ui", [
         }
 
         for (let child of json.children) {
-            if (child.type === constants.QUESTION_TYPE || child.type === constants.GROUP_TYPE || child.type === constants.REPEAT_TYPE) {
```

Comment:
I was surprised this constant wasn't deleted altogether, probably because there's one remaining reference, is that intentional? https://github.com/dimagi/commcare-hq/blob/e5ec4a4ef786e2efc2fc93f6c16e1c42389b76ee/corehq/apps/cloudcare/static/cloudcare/js/form_entry/utils.js#L242

Reply:
Thanks for pointing that out. Will need to test that.

Reply:
That was indeed a bug. Fixed with https://github.com/dimagi/commcare-hq/pull/34763/commits/5e019c589feea2b8b46ba3b73e0653cb49f7ccb1